server/channels/api4/plugin_test.go

```diff
@@ -1652,6 +1635,59 @@ func TestInstallMarketplacePlugin(t *testing.T) {
 		require.Nil(t, manifest)
 		assert.True(t, requestHandled)
 	}, "verify EnterprisePlugins is true for E20")
+}
+
+func TestInstallMarketplacePluginPrepackagedDisabled(t *testing.T) {
```

Comment:
Factored TestInstallMarketplacePluginPrepackagedDisabled into a separate method to avoid nested test helpers. Nested test helpers spin up duplicate servers and can cause issues if we rely on the expected singleton nature of the main helper.